backend/src/controllers/sessionsController.js

```diff
@@ -4,6 +4,6 @@ export const sessionsController = {
   post( req, res) {
     const user = req.body;
     const token = createToken(user);
-    res.header('auth_token', token).sendStatus(200);
+    res.json({ Token : token}).sendStatus(200);
```

Comment:
@LLaszlo89 json fieldnames should start with lowercase letters, `json()` method is already sending a response, calling `sendStatus` after is not really good
```suggestion
    res.status(200).json({ token : token});
```

but actually 200 is the default, so that's not even needed.